src/lib/release.ts

```diff
@@ -27,7 +27,15 @@ interface releases {
 }
 
 // Get release info of a certain verion of ORAS CLI
-export function getReleaseInfo(version: string) {
+export function getReleaseInfo(version: string, url: string, checksum: string) {
+    // if customized ORAS CLI link is provided
+    if (url && checksum) {
+        return {
+            checksum: checksum, 
+            url: url
+        } 
+    }
```

Comment:
What if `url` is provided but `checksum` is not provided? What's the proper error message?

Reply:
What will happen when `checksum` is provided if `version` is provided but `url` is not provided?

Reply:
What if all of three parameters are provided?